app/components/sdg/goals/index_component.rb

```diff
@@ -1,10 +1,11 @@
 class SDG::Goals::IndexComponent < ApplicationComponent
-  attr_reader :goals, :phases
+  attr_reader :goals, :phases, :header
   delegate :link_list, to: :helpers
 
-  def initialize(goals, phases)
+  def initialize(goals, phases, header)
```

Comment:
Since the `header` is listed before the `phases`, the order of the parameters is a bit unintuitive :thinking:.

How about using named parameters so the order doesn't matter?

```suggestion
  def initialize(goals, header:, phases:)
```